app/assets/javascripts/govuk_publishing_components/lib/header-navigation.js

```diff
@@ -22,12 +22,12 @@
         if (targetClass.indexOf('js-visible') !== -1) {
           target.setAttribute('class', targetClass.replace(/(^|\s)js-visible(\s|$)/, ''))
           if (window.GOVUK.analytics && window.GOVUK.analytics.trackEvent) {
-            window.GOVUK.analytics.trackEvent('headerClicked', 'searchClosed', { label: 'none' })
+            window.GOVUK.analytics.trackEvent('headerClicked', 'menuClosed', { label: 'none' })
```

Comment:
What's this change doing here?

Reply:
By which I mean, I don't want to know what this is doing, more why it's in this PR. Looks like it came from here https://github.com/alphagov/govuk_publishing_components/pull/2139 Are you in need of some rebase magic?

Reply:
Yes! Done.